src/main/java/at/hannibal2/skyhanni/config/features/mining/BreakDownConfig.java

```diff
@@ -0,0 +1,28 @@
+package at.hannibal2.skyhanni.config.features.mining;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorKeybind;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import org.lwjgl.input.Keyboard;
+
+public class BreakDownConfig {
```

Comment:
please add the prefix mining to the class name, and lowercase the D in breakdown